packages/maskbook/src/plugins/BarnBridge/constants.ts

```diff
@@ -0,0 +1,28 @@
+import { escapeRegExp } from 'lodash-es'
+
+export const BARNBRIDGE_PLUGIN_ID = 'com.barnbridge'
+
+export const BASE_URL = 'barnbridge.com'
+export const APP_URL = 'https://app.barnbridge.com/'
+export const API_URL = 'https://api.barnbridge.com/'
+export const SY_URL = 'smart-yield/'
```

Comment:
Not a valid URL?

Reply:
I can rename it to `SY_URL_FRAGMENT` since it's not technically a valid url